docker-compose.yml

```
@@ -64,9 +65,25 @@ services:
       - "9200:9200"
     mem_limit: 2g
 
-  # -- tools
   ralph:
     image: fundocker/ralph
+    ports:
+      - "${RALPH_RUNSERVER_PORT:-8100}:${RALPH_RUNSERVER_PORT:-8100}"
+    command:
+      - ralph
+      - "-v"
+      - DEBUG
+      - runserver
+      - "-b"
+      - "es"
+    environment:
+        RALPH_RUNSERVER_PORT: "${RALPH_RUNSERVER_PORT:-8100}"
+        RALPH_BACKENDS__DATABASE__ES__HOSTS: "${WARREN_ES_HOSTS:-http://elasticsearch:9200}"
+        RALPH_BACKENDS__DATABASE__ES__INDEX: "${WARREN_ES_INDEX:-statements}"
+        RALPH_AUTH_FILE: "src/backend/ralph_auth.json"
+    volumes:
+    - ./src/backend/ralph_auth.json:/app/src/backend/ralph_auth.json
```

It would be nice to automate the creation of the `.ralph/auth.json` file using the `compose run ralph auth` command in the Makefile, and then mount it as follow:

```suggestion
    - .ralph/auth.json:/var/run/ralph/auth.json
```